modules/plugin/src/main/scala/com/fortysevendeg/hood/plugin/SbtHoodPlugin.scala

```diff
@@ -96,6 +136,40 @@ object SbtHoodPlugin extends AutoPlugin with SbtHoodDefaultSettings with SbtHood
 
   }
 
+  private def submitResultsToGitHub[F[_]](
+      benchmarkResult: List[BenchmarkComparisonResult],
+      previousPath: File,
+      currentPath: File,
+      params: GitHubParameters)(
+      implicit L: Logger[F],
+      S: Sync[F]): EitherT[F, GHException, Unit] = {
+    val gh = GithubService.build[F]
```

Comment:
Maybe we could move this to a `lazy` val in the `SbtHoodPlugin` object initialized over `IO`

Reply:
Thanks for the suggestion! Done 😄